middleware/proxy/proxy_test.go

```diff
@@ -299,17 +299,17 @@ func Test_Proxy_Timeout_Slow_Server(t *testing.T) {
 	t.Parallel()
 
 	_, addr := createProxyTestServer(t, func(c fiber.Ctx) error {
-		time.Sleep(2 * time.Second)
+		time.Sleep(300 * time.Millisecond)
```

Comment:
This a big reduction in sleep time?

Reply:
the reduction for the sleep was intended 
and should make no difference